internal/services/container/testdata/namespace-vpc-integration.cassette.yaml

```diff
@@ -0,0 +1,2904 @@
+        headers:
+            Content-Length:
+                - "980"
+            Content-Security-Policy:
+                - default-src 'none'; frame-ancestors 'none'
+            Content-Type:
+                - application/json
+            Date:
+                - Thu, 12 Jun 2025 15:37:44 GMT
+            Server:
+                - Scaleway API Gateway (fr-par-2;edge02)
+            Strict-Transport-Security:
+                - max-age=63072000
+            X-Content-Type-Options:
+                - nosniff
+            X-Frame-Options:
+                - DENY
+            X-Request-Id:
+                - ce648131-b9ce-4a0d-a5d7-b6e5bfdf43d6
+        status: 200 OK
+        code: 200
+        duration: 93.333914ms
+    - id: 18
+      request:
+        proto: HTTP/1.1
+        proto_major: 1
+        proto_minor: 1
+        content_length: 219
+        transfer_encoding: []
+        trailer: {}
+        host: api.scaleway.com
+        remote_addr: ""
+        request_uri: ""
+        body: '{"privacy":"unknown_privacy","protocol":"unknown_protocol","secret_environment_variables":null,"http_option":"unknown_http_option","sandbox":"unknown_sandbox","private_network_id":"defa2ddd-f588-426e-b9a8-9c9a1fa7e231"}'
+        form: {}
+        headers:
+            Content-Type:
+                - application/json
+            User-Agent:
+                - scaleway-sdk-go/v1.0.0-beta.7+dev (go1.24.1; linux; amd64) terraform-provider/develop terraform/terraform-tests
+        url: https://api.scaleway.com/containers/v1beta1/regions/fr-par/containers/484195a3-2d06-4e91-97d1-7bb2012cf139
+        method: PATCH
+      response:
+        proto: HTTP/2.0
+        proto_major: 2
+        proto_minor: 0
+        transfer_encoding: []
+        trailer: {}
+        content_length: 121
+        uncompressed: false
+        body: '{"message":"Application can''t be attached to private network, vpc integration must be activated on its parent namespace"}'
+        headers:
+            Content-Length:
+                - "121"
+            Content-Security-Policy:
+                - default-src 'none'; frame-ancestors 'none'
+            Content-Type:
+                - application/json
+            Date:
+                - Thu, 12 Jun 2025 15:37:44 GMT
+            Server:
+                - Scaleway API Gateway (fr-par-2;edge02)
+            Strict-Transport-Security:
+                - max-age=63072000
+            X-Content-Type-Options:
+                - nosniff
+            X-Frame-Options:
+                - DENY
+            X-Request-Id:
+                - d3b7c8ba-8e96-476b-970e-27ba92704224
+        status: 400 Bad Request
```

Comment:
I'm not sure we should merge that. This request does not look ok

Reply:
I wanted to test the fact that legacy namespaces are not compatible with VPC and that activating the feature in the namespace should recreate it with ForceNew, therefore changing its ID.
I can remove the 2nd step if we prefer not to test whether the error occurs, but only the resource recreation.